geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/protobufs/TupleProtoBuf.scala

```diff
@@ -50,6 +74,17 @@ trait TupleProtoBuf {
         multibandTileProtoBufCodec.decode(message.tiles.get))
   }
 
+  implicit def tupleSpatialKeyImageProtoBufCodec =
+    new ProtoBufCodec[(SpatialKey, Array[Byte]), ProtoTuple] {
+    def encode(tuple: (SpatialKey, Array[Byte])): ProtoTuple =
+      ProtoTuple(
+        spatialKey = Some(spatialKeyProtoBufCodec.encode(tuple._1)),
+        imageBytes = ByteString.copyFrom(tuple._2))
```

Comment:
I chose `imageBytes` because it's the `bytes` of both the PNG's and GeoTiffs which are both images. I could think of something else, though.

Which `bytes` are you referring to? `bytes` in ProtoBuf is optional.